section2/reflection.md

```diff
@@ -1,15 +1,26 @@
+show a growth mindset currently by seeking out challenges, at this point in the
+program due to working a full time job I have been trying to just get everything
+done. I haven’t had the time to fully jump in.
 
-3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
+3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning? it felt good to get in a groove of learning to find an answer for something on my own. First I go back and check the notes provided, if I do not see anything then I begin to look on google for instructions.  So far I have found I am able to answer more all my own questions with a little work
 
-4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
+4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.Conditional statements evaluate to true or false.this can be done by using a number of different operators- strictly equal, not equal, greater than, etc.  A web example would be someone signing into a banking website and selecting from a list of languages
 
-5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do. Use either && or || i.e. logical AND or logical OR operator to connect two or more conditions inside a if statement. you just need to make sure your conditions are wrapped in parenthesis
 
-6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code? if else statements
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions? i still feel a little bit fuzzy making sure that I am writing out my coding correct for if else statements.  I think this should improve with more practice and getting a better grasp of the logic behind it
```

Comment:
Don't worry too much if this doesn't feel familiar yet, you will get a lot of practice with if statements in the program and after. Your if statements look great.